verl/third_party/sglang/parallel_state.py

```diff
@@ -9,16 +9,13 @@
 
 import torch
 import torch.distributed
-import vllm.distributed.parallel_state as ps
-from vllm.distributed.parallel_state import (
+import sglang.srt.distributed.parallel_state as ps
+from sglang.srt.distributed.parallel_state import (
     get_pp_group,
     get_world_group,
     init_distributed_environment,
     init_model_parallel_group,
 )
-from vllm.logger import init_logger
-
-logger = init_logger(__name__)
 """
 This version is strongly tied with Megatron to implement HybridEngine and weight sharing between vllm and Megatron.
```

Comment:
BTW i think the documentation requires updates too~
But it's okay to do it in upcoming PRs